free-2.md

```diff
@@ -10,34 +10,46 @@ good way to meet their needs.
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
+The need is for professors and students to be able to communicate problem sets and solutions more easily and through technology. I would be helping both teachers and students. Right now, for a student that does their problem sets in latex, someone needs to pick up the problem set in hard copy or print it out, then write it up in latex, and then print out the typesetted version to hadn it in. With my idea this same student would be able to receive the problem set as latex, do it in latex, and then submit it in latex (potentially with a wrapper around latex to make it a bit easier?)
 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
+A DSL is appropriate because it would work with latex and could be written by both teachers and students to communicate uniformly. It addresses the need by removing the need to print!
```

Comment:
I'm confused. What is the actual language part of it? This seems like people just send latex files to one another.